parsing/longident.mli

```diff
@@ -24,3 +24,8 @@ val flatten: t -> string list
 val unflatten: string list -> t option
 val last: t -> string
 val parse: string -> t
+
+[@@@ocaml.doc{|
+To print a longident, see {!Pprintast.longident}, using
+{!Format.asprintf} to convert to a string.
+|}]
```

Comment:
Was there a problem with a documentation comment here (`(** ... *)`) ?

Reply:
I wasn't sure that I would get the right comment-by-itself behavior with just newlines, so I thought that this more explicit syntax would be better. Do you think I should change back?

Reply:
I think so: I just realized that the right attribute was `ocaml.text` and not `ocaml.doc`. A newline should be fine  (and checking with `-dsource` works well).

Reply:
Good catch... I just changed it.